sim-lib/src/lib.rs

```diff
@@ -61,7 +61,7 @@ pub struct ClnConnection {
 }
 
 #[derive(Debug, Serialize, Deserialize, Clone)]
-pub struct Config {
+pub struct SimDetails {
```

Comment:
I don't love this name because writing "run simulation with details" or "run simulation with simulation file" in a readme doesn't _really_ make sense to me. 

gpt says setup.json or params.json but I don't know those are better.

Reply:
I'm happy with SimParams if you want to call them simulation parameters

Reply:
Feelings aren't strong enough for another round, happy to go in as-is.